src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,83 @@
+package duke;
+
+import duke.command.UserCommand;
+import duke.exceptions.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
+import javafx.application.Application;
+import javafx.scene.layout.Region;
+import javafx.stage.Stage;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.stage.Stage;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+
+/**
+ * Duke program.
+ */
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    private ScrollPane scrollPane;
+    private VBox dialogContainer;
+    private TextField userInput;
+    private Button sendButton;
+    private Scene scene;
+
+    private Image user = new Image(this.getClass().getResourceAsStream("/images/DaUser.png"));
+    private Image duke = new Image(this.getClass().getResourceAsStream("/images/DaDuke.png"));
```

Comment:
Should this be userImage and dukeImage so that it's consistent with the "MainWindow" file, and to avoid confusion between a Duke instance (of class Duke) and duke image (of class Image)?